bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClient.java

```diff
@@ -80,8 +80,7 @@ CheckAndMutateRowResponse checkAndMutateRow(CheckAndMutateRowRequest request)
    * Sample row keys from a table, returning a Future that will complete when the sampling has
    * completed.
    */
-  ListenableFuture<ImmutableList<SampleRowKeysResponse>> sampleRowKeysAsync(
-      SampleRowKeysRequest request);
+  ListenableFuture<List<SampleRowKeysResponse>> sampleRowKeysAsync(SampleRowKeysRequest request);
```

Comment:
The response is set as List<RESPONSE> in readAsync().
ListenableFuture<List<RESPONSE>> readyAsync(REQUEST request);

Usually we don't have to expose ImmutableList even though the list is immutable.